src/applications/static-pages/sharable-links/sharableLink.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const copyToUsersClipBoard = dataEntityId => {
+  const input = document.createElement('input');
+
+  const copyUrl = window.location.href.replace(window.location.hash, '');
+  input.setAttribute('value', `${copyUrl}#${dataEntityId}`);
+  document.body.appendChild(input);
+  input.select();
+  document.execCommand('copy');
+  document.body.removeChild(input);
+};
```

Comment:
Can we add a blank line here for readability?

Reply:
our eslint config don't like it :( 
![Uploading Screen Shot 2021-04-13 at 11.58.02 AM.png…]()